src/publicatie_tabellen/management/commands/pgdump.py

```diff
@@ -26,25 +27,32 @@ def handle(self, *args, **options):
                 latest_change = ChangesLog.objects.order_by('-changed_at').first()
                 if latest_change.changed_at <= updated.updated_at:
                     logger.info("No changes in tables")
+                    changed = False
                     return  # Stop de transactie
                 
                 PublishFunction.run_all_publish_tables()
                 updated.save()
-            
+                changed = True
+
+        except Exception as e:
+            changed = False
```

Comment:
Is het niet handiger om hier de exceptie te loggen en dan een return te doen?
Het bijhouden van een `changed` variabele is dan helemaal niet meer nodig